changedetection.py

```diff
@@ -12,13 +12,19 @@
 import os
 
 def sigterm_handler(sig, frame):
+    print("Catch - SIGTERM")
     raise SystemExit
+def sigint_handler(sig, frame):
+    print("Catch - CTRL+C")
+    # Trigger SystemExit. But SIGINT already sent to the parse_process.
```

Comment:
This is the detailed SIGINT mechanism. 
https://stackoverflow.com/a/31907519/20307768
https://unix.stackexchange.com/a/403605